lib/deprecation-cop-status-bar-view.coffee

```diff
@@ -34,13 +34,16 @@ class DeprecationCopStatusBarView extends View
 
   getDeprecatedSelectorCount: ->
     @deprecatedSelectorCount ?= getSelectorDeprecationsCount()
+  
+  getDeprecatedCallCount: ->
+    (deprecation.getStacks().length for deprecation in Grim.getDeprecations()).reduce(((a, b) -> a + b), 0);
```

Comment:
Just wondering if this might be more straight-forward as:

```coffee
Grim.getDeprecations().map((d) -> d.getStacks().length).reduce(((a, b) -> a + b), 0)
```

Let me know what you think. I'm cool with it if you want to keep it the current way but just a heads up, there is a `;` at the end of the line.